flixel/ui/FlxButton.hx

```diff
@@ -177,6 +177,11 @@ class FlxTypedButton<T:FlxSprite> extends FlxSprite implements IFlxInput
 	public var pressed(get, never):Bool;
 	public var justPressed(get, never):Bool;
 	
+	/**
+	 * We cast label to a FlxSprite for internal operations to avoid Dynamic casts in C++
+	 */
+	private var _spriteLabel:FlxSprite;
```

Comment:
Yes. Unless we want to turn FlxTypedButton into a full @:generic -- there we some concerns about that (see MSGhero below), so just using an underlying _spriteLabel avoids the Dynamic casts in a minimally invasive way. Unfortunately there's no way for me to use the same minimal approach on FlxPool -- it's @:generic or nothing.